modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+* Node configuration, which includes scripts for provisioning AWS and Azure infrastructure, preparing each node, bootstrapping a new Runtime Fabric, adjusting operating system limits, and performing jobs to clean up system resources.
++
+Node configuration upgrades are delivered as an updated install script version using the *Runtime Fabric downloads* page in Runtime Manager.
++
+[NOTE]
+If you are upgrading the node configuration with updated install scripts, the latest version of the cluster-level components (installer package) is automatically downloaded and used.
 
-## Downloads page
+It is important to run with the most current versions of the cluster-level (installer package) and `rtfctl` components.
 
-The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
+== Determine the Component Versions You Are Using
 
-The *Runtime Fabric downloads* page also provides an installer link.
+Run the following command:
+```
+/opt/anypoint/runtimefabric/rtfctl version
+```
+
+Output includes:
+
+* The `rtfctl` component version in the `rtfctl` field
+* The installer package component version in the `appliance` field
+* The Runtime Fabric component version in the `agent` field
 
-If an appliance upgrade is available, a code snippet for performing the upgrade is provided, as shown in the following example:
+For example:
 ```
-rtfctl appliance upgrade --url https://<upgrade-package-information>
+[root@ip-10-0-245-74 /]# /opt/anypoint/runtimefabric/rtfctl version
+COMPONENT    VERSION                  
+rtfctl       0.2.96+2868e14           
+agent        1.5.4                    
+appliance    1.1.1581641679-5884bce   
+kubernetes   1.13.12
 ```
 
+[NOTE]
+The Runtime Fabric component and cluster-level component (appliance) versions that are in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
+
+== Determine the Most Recent Component Versions Available
+The most recent component versions are displayed in the *Downloads* page, located in Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
+
+You can also review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+
+=== Apply the Upgrade
+Before upgrading your production environment, perform the following steps on a nonproduction Runtime Fabric to verify compatibility with your environment.
+
+. Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+. Provision additional worker VMs as needed to maintain availability of applications:
```

Comment:
Because there is just one step, change substep to a followup para:
```suggestion
. Provision additional worker VMs as needed to maintain the availability of applications.
```